.github/workflows/release.yml

```diff
@@ -42,9 +42,47 @@ jobs:
           channel: "stable"
       - name: Run CI
         run: bash ./scripts/run_ci_tasks.sh -i
+  upload-docs:
+    runs-on: macos-latest
+    needs: [analyze, android, ios]
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '12.x'
+      - name: Flutter action
+        uses: subosito/flutter-action@v1
+        with:
+          flutter-version: '1.12.13+hotfix.8'
```

Comment:
why is this version different?

Reply:
It's the last stable version. I fixed the version because I didn't find another way to get the path of the FLUTTER_ROOT. 
 `FLUTTER_ROOT: /Users/runner/hostedtoolcache/flutter/1.12.13-hotfix.8-stable/x64`